src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -537,6 +537,12 @@ services:
 
   PrestaShopBundle\Form\Admin\Type\AmountCurrencyType:
 
+  PrestaShopBundle\Form\Admin\Sell\Catalog\AttributeGroupType:
+    public: true
```

Comment:
im pretty sure it can also be autowired and private

Reply:
Romeved public true, but is autwire needed? there defaults autowire: true at the top of the file

Reply:
oh if there is default autowire:true then its all good (didn't see that)